hist/histpainter/src/THistPainter.cxx

```diff
@@ -8511,129 +8512,123 @@ void THistPainter::PaintStat(Int_t dostat, TF1 *fit)
    }
    if (print_name)  stats->AddText(fH->GetName());
    if (print_entries) {
-      if (fH->GetEntries() < 1e7) snprintf(t,100,"%s = %-7d",gStringEntries.Data(),Int_t(fH->GetEntries()+0.5));
-      else                        snprintf(t,100,"%s = %14.7g",gStringEntries.Data(),Float_t(fH->GetEntries()));
-      stats->AddText(t);
+      if (fH->GetEntries() < 1e7) tt = TString::Format("%s = %-7d",gStringEntries.Data(),Int_t(fH->GetEntries()+0.5));
+      else                        tt = TString::Format("%s = %14.7g",gStringEntries.Data(),Float_t(fH->GetEntries()));
+      stats->AddText(tt.Data());
```

Comment:
As before, better use `tt.Form()` instead of `tt = TString.Format()`

Reply:
in that case that's every where... not only here ... ?

Reply:
Yes, tt.Form() potentially can reuse already allocated memory in TString, therefore it is preferable

Reply:
Ok. I will revisit this PR.